tests/test_wave_03.py

```diff
@@ -54,13 +54,10 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
This is a good start. One thing to think about is how is this different than the previous test? Both of them should create a list that includes `INTRIGUE_3`, `HORROR_1` and `FANTASY_4`. The difference is that this one should is testing to make sure a duplicate `INTRIGUE_3` doesn't show up. A slightly more appropriate assert would be to check and see if the output list looks the way we're expecting! 
```suggestion
    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3] 
```